### ISSUE TITLE
SessionStorage broken after loggin in

### ISSUE THREAD

**Issue by inyono** [serlo/frontend#444]
After logging in, Next.js throws an error (at least locally).
To reproduce:

- Open http://localhost:3000
- Clear Cookies & Session Storage for good measure ;)
- Reload http://localhost:3000
- Log in
- Notice that Next.js throws an error:
```
SyntaxError: JSON.parse: unexpected character at line 1 column 2 of the JSON data
```
- Notice that `___loggedInData_de` in Session Storage has a non-serialized value: "[object Object]" which is causing the error.

**Comment by inyono** [comment]
More specifically: This line https://github.com/serlo/frontend/blob/5582d3acc88d3e4b9c72292c82766a4ca0e9bd8a/src/pages/%5B%5B...slug%5D%5D.tsx#L145 is setting the session storage in an incorrect way.

**Comment by elbotho** [comment]
Oha, that is new :)

**Comment by Entkenntnis** [comment]
fixed

**Comment by inyono** [comment]
Wow, that was a fast hotfix 🎉. Agile AF!!! 😄

**Comment by inyono** [comment]
Fixed via c56a6fbff079581b3d63f7296fe46afa836653c6
